src/Raven.Client/Documents/Indexes/Counters/AbstractCountersIndexCreationTask.cs

```diff
@@ -89,6 +89,11 @@ public override CountersIndexDefinition CreateIndexDefinition()
                     builder.AddMap(_map.Counter, _map.Map);
             }
 
+            if (SearchEngineType.HasValue)
+            {
```

Comment:
Are we missing `CompoundFieldsStrings` in the builder?

Reply:
Let's handle this as a separate issue

Reply:
https://issues.hibernatingrhinos.com/issue/RavenDB-22529/Add-CompoundFieldsStrings-to-Counter-and-TimeSeries-abstract-creation-tasks